### ISSUE TITLE
Unable to convert colorspace

### ISSUE THREAD

**Issue by wyred** [avalanche123/Imagine#374]
I have a jpg in CMYK colorspace.

Here's a simplified version of my code

```
$image = $imagine->open($source_file);
$image->crop($crop_point, $square_box)
    ->resize($resize_box)
    ->strip()
    ->usePalette(new Imagine\Image\Palette\RGB())
    ->save($destination_file);
```

But when I open the resulting file in photoshop, it's still in CMYK.

**Comment by edigu** [comment]
@romainneutron is there any known issue exist which related to mac os x? I have same issue on os x.

**Comment by trsteel88** [comment]
@edigu @romainneutron I'm having the same issue.

The dump says "rbg". However, saving the image does not result in an RBG image. After saving and downloading the file it is still CMYK.

**Comment by trsteel88** [comment]
Using the command line it seems to work though.

![screen shot 2015-11-11 at 10 49 13 pm](https://cloud.githubusercontent.com/assets/869933/11091485/8d296f8a-88c6-11e5-89c8-78123cc31911.png)

So I am starting to this it is related directly to this library.

**Comment by trsteel88** [comment]
Anyone?

**Comment by ausi** [comment]
I think this bug gets fixed by #564

**Comment by ausi** [comment]
I had the same issue, the problem was caused by a missing ImageMagick delegate called `lcms` or `lcms2`. You can see if this is installed on your system by calling `convert -list configure | grep DELEGATES` on the command line, if `lcms` doesn’t show up in this list you have to reinstall ImageMagick with the `--with-lcms2` or `--with-lcms` flag.
